pkg/cluster/dao/dao.go

```diff
@@ -251,6 +251,14 @@ func (d *dao) List(ctx context.Context,
 				statement = statement.Where("c.template = ?", v)
 			case common.ClusterQueryByRelease:
 				statement = statement.Where("c.template_release = ?", v)
+			case common.ClusterQueryIsFavorite:
+				isFavoriteInter := query.Keywords[common.ClusterQueryIsFavorite]
+				isFavorite := isFavoriteInter.(bool)
+				if isFavorite {
+					statement = statement.Where("c.id in (select resource_id from tb_collection where resource_type = 'cluster')")
```

Comment:
Does user_id not need to be included in the query expression?

Reply:
fixed